src/pykiso/cli.py

```diff
@@ -20,12 +20,13 @@
 
```

Comment:
Update the RobotLoader and make it run for the RobotFramework

Reply:
Not necessary at the end